src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from 'react'
+import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
+import * as api from '../utils/api'
+import './comments.css'
+
+const DeleteComment = ({comment_id, renderKey, setRenderKey}) => {
+
+  const [isDeleted, setIsDeleted] = useState(false)
+    const handleClick = (comment_id) => {
+      setIsDeleted(true)
+      api.deleteComments(comment_id).then(() => {
```

Comment:
What happens if this api request fails?

Reply:
I'm thinking of doing my last ticket for error handling, and once that gets approved I'm planning to work on creating user and implementing logging in/out functions